climada/hazard/tc_tracks_synth.py

```diff
@@ -143,25 +195,47 @@ def calc_perturbed_trajectories(tracks,
                                  max_dspeed_rel, max_ddirection, rand)
                    for track, rand in zip(tracks.data, random_vec)]
 
+    cutoff_track_ids_tc = [x[1] for x in new_ens]
+    cutoff_track_ids_tc = sum(cutoff_track_ids_tc, [])
+    cutoff_track_ids_ts = [x[2] for x in new_ens]
+    cutoff_track_ids_ts = sum(cutoff_track_ids_ts, [])
+    if len(cutoff_track_ids_tc) > 0:
```

Comment:
Thanks for updating the logging levels. 

One more question here: Suppose I generated tracks using this synthetic track generation method, and I get some tracks that where cut out at TC_category > 1. What should I do now? I could remove the track from the set, but this might screw up the frequency right? Otherwise I could rerun and hope to be lucky?